appinventor/common/src/com/google/appinventor/common/utils/StringUtils.java

```diff
@@ -346,4 +348,22 @@ public static int countMatches(String str, String sub) {
   public static boolean isNullOrEmpty(final String text) {
      return text == null || text.isEmpty();
   }
+
+  /**
+   * Given an array of bytes, returns its hexadecimal representation.
+   * @param buffer array of bytes to represent
+   * @return hexadecimal representation of bytes
+   */
+  // Solution from https://stackoverflow.com/a/9855338
```

Comment:
I had to use a different solution for this method, as moving the method to `StringUtils` was failing to compile at `YaClientApp` with an error stating that `String.format` was not available (which version is it using to compile it?).

I think it's not yet ready, as right now `AndroidRuntime` does not include `CommonUtils.jar`, but will test it tomorrow.

Reply:
> I think it's not yet ready, as right now AndroidRuntime does not include CommonUtils.jar, but will test it tomorrow.

Solved.